docs/joint_models_estimating_parameters.rst

```diff
@@ -10,7 +10,224 @@ Creating a global hierarchical model
+        m : float, optional
+            First shape parameter. Defaults to self.m.
+        c : float, optional
+            The second shape parameter. Defaults to self.k.
+        lambda_: float, optional
+            The reciprocal scale parameter . Defaults to self.lambda_.
+
+        """
+
+        scipy_par = self._get_scipy_parameters(m, c, lambda_)
+        return sts.gengamma.pdf(x, *scipy_par)
+
+Another important function is to draw random samples from the distribution. Hence, every statistical function in
+virocon must provide a draw_sample function:
+
+.. code-block:: python
+
+    def draw_sample(self, n, m=None, c=None, lambda_=None):
+        scipy_par = self._get_scipy_parameters(m, c, lambda_)
+        rvs_size = self._get_rvs_size(n, scipy_par)
+        return sts.gengamma.rvs(*scipy_par, size=rvs_size)
+
+Given a data set is available, a user might want to fit a generalized gamma distribution to these data. The fit method
+returns estimates of the shape and scale parameters from data. The default estimation method is maximum likelihood
+estimation (MLE), which is why in virocon all statistical distributions are equipped with a function to fit a
+distribution to a data set by means of the MLE. Here, the user should be able to hold some parameters fixed to certain
+values by passing in keyword arguments.
+
+.. code-block:: python
+
+    def _fit_mle(self, sample):
+        p0 = {"m": self.m, "c": self.c, "scale": self._scale}
+
+        fparams = {"floc": 0}
+
+        if self.f_m is not None:
+            fparams["fshape1"] = self.f_m
+        if self.f_c is not None:
+            fparams["fshape2"] = self.f_c
+        if self.f_lambda_ is not None:
+            fparams["fscale"] = 1 / (self.f_lambda_)
+
+        self.m, self.c, _, self._scale = sts.gengamma.fit(
+            sample, p0["m"], p0["c"], scale=p0["scale"], **fparams
+        )
+
+    def _fit_lsq(self, data, weights):
+        raise NotImplementedError()
+
+**3. Use new distribution**:
+
+The above-described steps all must be implemented in the distributions.py file of virocon.
+In order to use the new implemented distribution, add the name of the new distribution into the variable _all_=[] below
+the imports.
+
+.. figure:: Use_new_dist.png
+    :scale: 50 %
+    :alt: Use new distribution in virocon.
+
+**4. Write automatic tests**:
+
+Before implementing the new distributions in virocon, we want to know, if the above-described steps and functions really
+perform as expected. Therefore, the most accurate test is to reconstruct a distribution from literature and compare the
+results. If the results match, we can have high certainty that we implemented the new distribution correctly. In
+general, every function of a class must be tested. To conduct automatic tests, virocon uses pytest. To be able to
+execute these tests automatically, the added test files for a new distribution must be attached to the file
+test_distributions.py.
```

Comment:
Just a comment: it is not necessary to implement the tests in test_distributions.py. Any other file in the same place that starts with test_ would be fine too. It is just sensible to put the tests there.